<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="24,1,12">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="5,9">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="9,1">
         <item>
          <widget class="QLabel" name="greenPic">
           <property name="text">
            <string>可回收
green</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="greenLabel">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4" stretch="9,1">
         <item>
          <widget class="QLabel" name="bluePic">
           <property name="text">
            <string>厨余垃圾
blue</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="blueLabel">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5" stretch="9,1">
         <item>
          <widget class="QLabel" name="redPic">
           <property name="text">
            <string>有害垃圾
red</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="redLabel">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6" stretch="9,1">
         <item>
          <widget class="QLabel" name="yellowPic">
           <property name="text">
            <string>其他垃圾
yellow</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="yellowLabel">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="infoLabel">
         <property name="text">
          <string>主板温度：
目前状态：</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="moviePlayer">
       <property name="text">
        <string>视频播放</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="3,9">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="nameLabel">
         <property name="text">
          <string>垃圾桶编号
1000</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="QRPic">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="hisList">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="plainText">
        <string>历史垃圾列表:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
